packages/manager/cypress/e2e/core/linodes/create-linode-with-firewall.spec.ts

```diff
@@ -179,4 +185,177 @@ describe('Create Linode with Firewall', () => {
     // Confirm toast notification should appear on Linode create.
     ui.toast.assertMessage(`Your Linode ${mockLinode.label} is being created.`);
   });
+
+  /*
+   * - Mocks the internal header to enable the Generate Compliant Firewall banner.
+   * - Confirms that Firewall is reflected in create summary section.
+   * - Confirms that outgoing Linode Create API request specifies the selected Firewall to be attached.
+   */
+  it('can generate and assign a compliant Firewall during Linode Create flow', () => {
+    cy.intercept(
+      {
+        middleware: true,
+        url: /\/v4(?:beta)?\/.*/,
+      },
+      (req) => {
+        // Re-add internal-only header
+        req.on('response', (res) => {
+          res.headers['akamai-internal-account'] = '*';
+        });
+      }
+    );
+
+    const linodeRegion = chooseRegion({ capabilities: ['Cloud Firewall'] });
+
+    const mockFirewall = firewallFactory.build({
+      id: randomNumber(),
+      label: randomLabel(),
+    });
+
+    const mockLinode = linodeFactory.build({
+      id: randomNumber(),
+      label: randomLabel(),
+      region: linodeRegion.id,
+    });
+
+    const mockTemplate = firewallTemplateFactory.build({
+      slug: 'akamai-non-prod',
+    });
+
+    mockCreateFirewall(mockFirewall).as('createFirewall');
+    mockGetFirewalls([mockFirewall]).as('getFirewall');
+    mockGetTemplate(mockTemplate).as('getTemplate');
+    mockCreateLinode(mockLinode).as('createLinode');
+    mockGetLinodeDetails(mockLinode.id, mockLinode);
+
+    cy.visitWithLogin('/linodes/create');
+
+    linodeCreatePage.setLabel(mockLinode.label);
+    linodeCreatePage.selectImage('Debian 11');
+    linodeCreatePage.selectRegionById(linodeRegion.id);
```

Comment:
```suggestion
    linodeCreatePage.selectRegionById(linodeRegion.label);
```
Not sure if this is applicable only for me, but all the tests seem to fail when I use `id` instead of `label`

Reply:
I was encountering this issue too, but it only occurs when running tests locally with the Gecko v2 GA flag enabled.

https://github.com/linode/manager/blob/6c9ad93851b2c22bf3c4b7a5964d3057d4145a37/packages/manager/src/components/RegionSelect/RegionSelect.tsx#L122-L124

Reply:
We may have to change all instances like this in the near future if GA becomes permanent and the flag is retired (cc @jdamore-linode)

Reply:
Hey @hkhalil-akamai, I can't run the tests locally right now because I'm running DC tests, but is this failing because Cypress is typing the region ID into a region select field and the desired region is not showing up? If so, I believe that's a Gecko bug (M3-8502) rather than an issue with the tests.